package.json

```diff
@@ -46,23 +46,23 @@
     "@genesislcap/foundation-reporting": "14.137.0",
     "@genesislcap/foundation-testing": "14.137.0",
     "@genesislcap/foundation-zero": "14.137.0",
-    "@mui/icons-material": "^5.11.0",
-    "@mui/material": "^5.11.6",
-    "classnames": "^2.3.2",
-    "dayjs": "1.10.7",
+    "@mui/icons-material": "^5.15.6",
+    "@mui/material": "^5.15.6",
+    "classnames": "^2.5.1",
+    "dayjs": "1.11.10",
     "docusaurus-gtm-plugin": "^0.0.2",
-    "docusaurus-plugin-matomo": "^0.0.6",
-    "esbuild": "0.17.19",
-    "esbuild-loader": "^3.0.1",
-    "fs-extra": "^11.1.0",
-    "mdx-mermaid": "1.3.2",
-    "mermaid": "^9.3.0",
+    "docusaurus-plugin-matomo": "^0.0.8",
+    "esbuild": "0.19.12",
+    "esbuild-loader": "^4.0.3",
```

Comment:
`esbuild-loader` is already specified in the `devDeps`

Reply:
good spot, must have been there for a long time, I just bumped versions
anyway sorted :)